gratipay/testing/browser.py

```diff
@@ -80,6 +81,32 @@ def has_element(self, selector, timeout=None):
         """
         return self.is_element_present_by_css(selector, timeout)
 
+    def confirming(self, answer):
+        """Return a context manager for working with confirm alerts cross-driver.
+        """
```

Comment:
Sadly, I think that Chrome 52 may actually break this. I'm almost positive it was working under Chrome 51. :-/